### PR TITLE
chore: Migrate MyFT email over to using www.ft.com rather than next.ft.com

### PR DESCRIPTION
**Pull request by matthew-andrews**
😄

### CODE REVIEW

File: functions/create-email/views/partials/excess.html [hunk 1]
```diff
@@ -133,7 +133,7 @@
 																									<tbody>
 																										<tr>
 																											<td valign="left" style="font-family: Arial, sans-serif;color: #9E2F50; font-size:16px; line-height:20px; padding-left:10px; padding-top: 10px; padding-bottom: 10px; text-align:left; font-weight:bold;">
-																												<a href="https://next.ft.com/myft#myft:notification:{{@root.type}}-email:feed:html" target="_blank" style="color: #9E2F50; text-decoration: none;"><span style="color: #9E2F50;">More new stories available in your feed in <i>my</i>FT</span></a><br>
+																												<a href="https://www.ft.com/myft#myft:notification:{{@root.type}}-email:feed:html" target="_blank" style="color: #9E2F50; text-decoration: none;"><span style="color: #9E2F50;">More new stories available in your feed in <i>my</i>FT</span></a><br>
```
Comment (matthew-andrews) [inline functions/create-email/views/partials/excess.html]: hm…… sed……

Comment (matthew-andrews) [inline functions/create-email/views/partials/excess.html]: look right >>>>>>>>>

**Comment by laurieboyes** [comment]
Rather you than me. We'll drop a card in our test column for Stephen to run his magic link checker thing on the preview 👍

**Comment by matthew-andrews** [comment]
`find . -type f -name '*.html' -exec sed -i '' "s/next.ft.com/www.ft.com/g" {} \;` 😉

**Pull request merged**